specs/ServiceProtocol.md

```diff
@@ -141,6 +148,7 @@ MessagePack uses different formats to encode values. Refer to the [MessagePack F
 - 5 - Message Type, indicating this is a `CloseConnection` message.
 - ConnectionId - A `String` encoding unique Id of the connection.
 - ErrorMessage - Optional `String` encoding error message.
+- LastSequenceId - A `Int64` encoding number indicates the last acked `ConnectionDataMessage` sequence id.
```

Comment:
>LastSequenceId [](start = 2, length = 14)

naming `LastAckId`?

Reply:
Actually we don't have an `Ack` action.
What we could ensure is we have delivered the message to the application, not `Ack`